src/pages/studyView/table/ClinicalTable.tsx

```diff
@@ -30,6 +30,16 @@ export default class ClinicalTable extends React.Component<IClinicalTableProps,
         super(props);
     }
 
+    static readonly defaultProps = {
+        width: 300
+    };
+
+    @computed
+    get columnWidth() {
+        let lastTwoColumns = [80, 60];
```

Comment:
could you abstract this to a method named add80Plus60() and then write a few tests for it?  ; )

Reply:
[freaking out]